packages/salesforcedx-vscode-apex/src/languageClientUtils/languageClientUtils.ts

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Refactored some of the original code into a singleton and moved it into a new `languageClientUtils` module.